docs/content/documentation/getting-started/directory-structure.md

```diff
@@ -49,3 +49,7 @@ and available variables.
 ## `themes`
 Contains themes that can be used for your site. If you are not planning to use themes, leave this directory empty.
 If you want to learn about themes, see the [themes documentation](@/documentation/themes/_index.md).
+
+## `public`
+Contains the built site from [`zola build`](@/documentation/getting-started/cli-usage.md#build).
```

Comment:
`zola serve` might create it as well. As mentioned in the issue, this folder can be specified in zola build/server.

I think I would just mention that by default a `public` folder might appear when running zola build/serve that contains some of the output

Reply:
Should be fine now? LMK if I should make any more adjustments